src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -34,6 +36,8 @@
 #include "openvino/runtime/properties.hpp"
 #include "transformations/convert_precision.hpp"
 
+const constexpr char* npuw_name_identifier = "NPUW_serialized_";
```

Comment:
The idea was to differentiate serialization route vs the default one. Since the API is fixed, I decided to handle it with model name.